src/sizzle.js

```diff
@@ -630,55 +654,55 @@ setDocument = Sizzle.setDocument = function( node ) {
 	// Check if getElementById returns elements by name
 	// The broken getElementById methods don't pick up programmatically-set names,
 	// so use a roundabout getElementsByName test
-	support.getById = assert(function( el ) {
+	support.getById = assert( function( el ) {
 		docElem.appendChild( el ).id = expando;
 		return !document.getElementsByName || !document.getElementsByName( expando ).length;
-	});
+	} );
 
 	// ID filter and find
 	if ( support.getById ) {
-		Expr.filter["ID"] = function( id ) {
+		Expr.filter.ID = function( id ) {
```

Comment:
Let's revert that to the `[ "ID" ]` syntax. Please override the proper rule from the jquery preset in the ESLint config.

This was done for some Google Closure Compiler compatibility and while we're not verifying it in any way, we'd rather avoid the risk at this point.

This applies to here & all other changes like that done on `Expr`.

Reply:
Ok, it's done a I have added `"dot-notation": "off"` to `.eslintrc-browser.json` to override that rule.